acsoo/addons.py

```diff
@@ -74,3 +75,25 @@ def addons_list_depends(ctx, exclude):
 
 
 addons.add_command(addons_list_depends, 'list-depends')
+
+
+@click.command()
+@click.option('--database')
+@click.option('--odoo-bin', 'odoo_bin', default='odoo',
+              type=click.Path(dir_okay=False, exists=True))
+@click.option('--odoo-config', 'odoo_config',
+              type=click.Path(dir_okay=False, exists=True))
+@click.option('--push', is_flag=True, default=False)
+@click.option('--branches', default='')
+@click.option('--user-name', 'user_name', default='')
+@click.option('--user-email', 'user_email', default='')
```

Comment:
I suggest prefixing these options with `git-` (git-push, git-usr-name, etc).

Reply:
Also, separate git-commit and git-push (useful for testing locally without pushing immediately)